README.md

```diff
@@ -57,6 +57,76 @@ them and then do:
 
     % git subrepo push ext/os-autoinst-common
 
+### Possible Workflows
+
+One can make changes to the subrepo directly from the host repo, but there are
```

Comment:
I think "host repo" has not been defined properly. I would vote for "superrepo" which would be the direct opposite of "subrepo". Also see https://answers.yahoo.com/question/index?qid=20080706235630AA5Eecl&guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZWNvc2lhLm9yZy9zZWFyY2g_cT1vcHBvc2l0ZStvZislMjJzdWIlMjImYWRkb249ZmlyZWZveCZhZGRvbnZlcnNpb249NC4wLjQ&guce_referrer_sig=AQAAAAGVqiCbnGrZSpgQowy4dQo-UPpqg7AmfBRUp8LnkXyRbxt26biINCQkoqkmE0TxoY7CBlDW86k71g1o4FhAFRKL_fTdLCG8CFOiDcFmxfZBf2t9mWwKrence5DOUraIhf0Cu_nqsVJsI1XrX7uM4Ijr3fj_dx2zejx5ik-Mc4Nb

Reply:
I'd suggest to change "top repo" in the previous section for consistency.

"Project repo" would work for me, too. All of those convey the point as far as I'm concerned.

Reply:
I changed it to "top repo"